sdks/python/container/run_generate_requirements.sh

```diff
@@ -30,6 +30,16 @@
 # You will need Python interpreters for all versions supported by Beam, see:
 # https://s.apache.org/beam-python-dev-wiki
 
+if ! [[ "$(expr substr $(uname -s) 1 5)" == "Linux" ]]; then
+  echo This script needs to be executed in a Linux environment.
```

Comment:
optional: update wiki page since devs mostly update it on their own systems (which is Mac).

Reply:
I believe the script already fails on Mac due to google-cloud-profiler or some other dependency not installable on Mac, that is why I added this. Can this script in current for be runnable on Mac?

Reply:
I believe Wiki already recommends  a linux machine.

Reply:
> I believe Wiki already recommends a linux machine.

looks like not. I definitely remember script failing on MacOS before.

Reply:
I've run this successfully from my (non-M1) mac before

Reply:
thanks,  added instructions for MacOS. @AnandInguva please help verify that is sufficient. You can check out the pr locally by smth like:

```
git fetch origin pull/29609/head:pr-29609                                           
git checkout pr-29609    
```

 Thanks!

Reply:
Do you need anything else from me here? asking in case if I missed something. As of now, this is not running on M1 Mac.

Reply:
i tried few things from stackoverflow and python-snappy github page but no luck so far.